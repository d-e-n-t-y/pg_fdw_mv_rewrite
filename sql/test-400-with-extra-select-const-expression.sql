explain (VERBOSE, COSTS OFF) select key, 'key=' || key, COUNT (value) from test_lb group by key;

select key, 'key=' || key, COUNT (value) from test_lb group by key;
