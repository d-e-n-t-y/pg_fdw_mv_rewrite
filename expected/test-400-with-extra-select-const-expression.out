explain (VERBOSE, COSTS OFF) select key, 'key=' || key, COUNT (value) from test_lb group by key;
                                 QUERY PLAN                                  
-----------------------------------------------------------------------------
 Foreign Scan
   Output: key, (('key='::text || key)), (count(value))
   Relations: Aggregate on (public.test_lb)
   Remote SQL: SELECT key, ('key='::text || key), count FROM public.test_mv1
(4 rows)

select key, 'key=' || key, COUNT (value) from test_lb group by key;
 key  | ?column? |  count  
------+----------+---------
 key1 | key=key1 |      10
 key2 | key=key2 | 1000000
(2 rows)

